### ISSUE TITLE
Misleading error messages on wallet locked

### ISSUE THREAD

**Issue by jholdstock** [decred/dcrwallet#1865]
Creating transactions fails when the wallet is locked, as expected, however the error messages are misleading:

```
dcrctl --testnet --wallet sendtoaddress TsUfz7s7XjRcYcTfRXjEkLKpw6EzGmhgBvp 10
-4: wallet.SendOutputs: no private key for default/1/3018
```

```
2020-10-07 11:30:39.929 [ERR] WLLT: Failed to sign revocation for ticket hash c1b7015f926cf0f54df70a72da451cf4cd1844800cbaf1116f2517a1686a3e40: txscript.SignTxOutput: transaction script fails to execute: no private key for default/0/211
```

A more helpful message would indicate that the wallet needs to be unlocked for these operations to succeed.

**Comment by davecgh** [comment]
100% agree.  The wallet used to explicitly tell you "wallet is locked -- use walletunlock to unlock it before signing" when at attempt to sign via RPC was made which was much more helpful.

**Comment by JoeGruffins** [comment]
Was fixed? I just tried and do see `-13: wallet or account locked; use walletpassphrase or unlockaccount first`

**Comment by davecgh** [comment]
I suspect it probably depends on what you're doing and how the condition is reached.

**Comment by jrick** [comment]
Fixed by #1947.
